packages/components/src/slot-fill/fill.js

```diff
@@ -51,28 +52,5 @@ function FillComponent( { name, children, registerFill, unregisterFill } ) {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [ name ] );
 
-	if ( ! slot || ! slot.node ) {
-		return null;
-	}
-
-	// If a function is passed as a child, provide it with the fillProps.
-	if ( typeof children === 'function' ) {
-		children = children( slot.props.fillProps );
-	}
-
-	return createPortal( children, slot.node );
+	return null;
 }
-
-const Fill = ( props ) => (
```

Comment:
I wonder why we had this in the first place.

Reply:
`FillComponent` used to be a class component, wrapped in a context consumer component, and then it was refactored to functional with hooks in #15541. But the PR kept the context consumer wrapper, it didn't migrate it to `useContext`.